security/models.py

```diff
@@ -142,32 +145,26 @@ def prepare_from_request(self, request):
         )
 
 
-class LoggedRequest(models.Model):
+class LoggedRequestStatus(ChoiceEnum):
```

Comment:
new enums

Reply:
how about putting them in an enums.py file?

Reply:
I don't see reason, there are not probability of circle imports